test/base/events_spec.js

```diff
@@ -173,18 +126,77 @@ describe('Events', function(){
   })
 
   it('still calls later handlers if one throws an exception', function(){
-    let secondHandlerCalled = false
     const handlerThatThrows = function() {
       throw new Error('Whoops')
     }
-    const handler = function() {
-      secondHandlerCalled = true
-    }
 
     this.events.on('clappr.any.event', handlerThatThrows)
-    this.events.on('clappr.any.event', handler)
+    this.events.on('clappr.any.event', this.callback)
     this.events.trigger('clappr.any.event')
 
-    expect(secondHandlerCalled).to.be.true
+    this.callback.should.have.been.calledOnce
+  })
+
+  it('when register, still call custom event', function(){
```

Comment:
The test description should be the actual behavior the component should present but in present tense.

Something like `triggers custom event when registered`. The same works for the following tests.